package.json

```diff
@@ -17,9 +17,9 @@
   "scripts": {
     "coverage": "istanbul cover node_modules/mocha/bin/_mocha -r test/fixtures/env.js test/**/*.js",
     "start": "node index",
-    "build": "npm run build:font-awesome && npm run build:grunt && npm run build:handlebars",
+    "build": "npm-run-all build:*",
```

Comment:
npm-run-all seems like a reasonable dependency at this point, but I can imagine simplifying things to a point where we can just use `&&` (or `&` for parallel) and drop a dependency. It's simple enough for now though, I think.